src/commands/login.rs

```diff
@@ -28,7 +28,7 @@ const SPIN_CLIENT_ID: &str = "583e63e9-461f-4fbe-a246-23e0fb1cad10";
 
 const DEFAULT_CLOUD_URL: &str = "https://cloud.fermyon.com/";
 
-/// Log into the Fermyon Platform.
+/// Log into the Fermyon Platform using credentials from ~/.docker/config.json.
```

Comment:
I don't think this makes sense at the present moment. `spin deploy` is still [using bindle API endpoints](https://github.com/fermyon/spin/blob/9cdf8aebf01f2dee6c2f1515d476f7d3642250c9/src/commands/deploy.rs#L350-L359), and when we implement OCI support, we won't be reading configuration from `~/.docker/config.json` IIRC.

Would you mind removing this until we make more progress on the OCI migration?

Reply:
@vdice do you have anything you want to add here?

Reply:
Ahh good catch @bacongobbler . This was meant to be attached to `registry login` but i had fatally goofed.

Reply:
Oops, I somehow misread this as a remove.  Yeah we should definitely not have this text.  Thanks for the catch @bacongobbler!

@fibonacci1729 I think this used to be true of `spin registry login` (not `spin login` itself), but I'm not sure if that's still the case.